Update node volume size details (#491)
Consensys/doc.zk-evm: docs/build-on-linea/run-a-node/volume-creation.mdx

- Open Terminal
- Use the `df -h` command to check the available disk space

> - Decide on the maximum size you want for the volume. We'll use 100GB for the purposes of this 
> explanation, but please refer to the [Geth](use-binary/#disk-space-geth) or [Besu](use-binary/#disk-space-besu)
> explanation to help choose the value.

Reviewer: ```suggestion
- Choose a maximum size for the volume. We'll use 100GB for this example, but refer to the [Geth](./use-binary.mdx#disk-space-geth) or [Besu](./use-binary.mdx#disk-space-besu) explanation to help determine the value.
```

Reviewer: Note, I've updated the format of the links here.

- Open Terminal
- Use the `df -h` command to check the available disk space

> - Decide on the maximum size you want for the volume. We'll use 100GB for the purposes of this 
> explanation, but please refer to the [Geth](use-binary/#disk-space-geth) or [Besu](use-binary/#disk-space-besu)
> explanation to help choose the value.

Reviewer: ```suggestion
- Choose a maximum size for the volume. We'll use 100GB for this example, but refer to the [Geth](./use-binary.mdx#disk-space-geth) or [Besu](./use-binary.mdx#disk-space-besu) explanation to help determine the value.
```

- Open Command Prompt as Administrator
- Use the `dir` command to check available disk space on the volume you want to create the disk image

> - Decide on the maximum size for the volume. We'll use 100GB for the purposes of this 
> explanation, but please refer to the [Geth](use-binary/#disk-space-geth) or [Besu](use-binary/#disk-space-besu)
> explanation to help choose the value.

Reviewer: ```suggestion
- Choose a maximum size for the volume. We'll use 100GB for this example, but refer to the [Geth](./use-binary.mdx#disk-space-geth) or [Besu](./use-binary.mdx#disk-space-besu) explanation to help determine the value.
```

- Open WSL
- Use the `df -h` command to check the available disk space

> - Decide on the maximum size you want for the volume. We'll use 100GB for the purposes of this 
> explanation, but please refer to the [Geth](use-binary/#disk-space-geth) or [Besu](use-binary/#disk-space-besu)
> explanation to help choose the value.

Reviewer: ```suggestion
- Choose a maximum size for the volume. We'll use 100GB for this example, but refer to the [Geth](./use-binary.mdx#disk-space-geth) or [Besu](./use-binary.mdx#disk-space-besu) explanation to help determine the value.
```
